draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -1079,6 +1101,40 @@ following Payload-specific RTP Feedback (PSFB) feedback.
   we have QUIC timestamps, we don't need to work around RTP timestamps offsets
   because we can use the QUIC timestamps to calculate network jitter.
 
+# Error Handling {#error-handling}
+
+The following error codes are defined for use when abruptly terminating streams,
+aborting reading of streams, or immediately closing RoQ connections.
+
+ROQ\_NO\_ERROR (0x????):
+: No error. This is used when the connection or stream needs to be closed, but
+there is no error to signal.
+
+ROQ\_INTERNAL\_ERROR (0x????):
+: An internal error has occured in the RoQ stack.
+
+ROQ\_PACKET\_ERROR (0x????):
+: Invalid payload format, e.g., length does not match packet, invalid flow id
+encoding, non-RTP on RTP-flow ID, etc.
+
+ROQ\_STREAM\_CREATION\_ERROR (0x????):
+: The endpoint detected that its peer created a stream that it will not accept.
+
+ROQ\_FRAME\_CANCELLED (0x????):
+: A receiving endpoint stopped reading a frame from a stream and requests new
+frames be sent on new streams using STOP\_SENDING or a sender notifies a
+receiver that retransmissions of a frame were stopped using RESET\_STREAM and
+new frames will be sent on new streams.
+
+ROQ\_UNKNOWN\_FLOW\_ID (0x????):
+: An endpoint was unable to handle a flow identifier, e.g., because it was not
+signalled or it does not support multiplexing using arbitrary flow identifiers.
```

Comment:
Perhaps 

> : An endpoint was unable to handle a flow identifier, e.g., because it was not
> signalled or **because the endpoint** does not support multiplexing using arbitrary flow identifiers.